tests/Aspire.Hosting.NodeJs.Tests/NodeAppFixture.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Testing;
+using Aspire.Hosting.Utils;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Aspire.Hosting.NodeJs.Tests;
+
+/// <summary>
+/// TestProgram with node and npm apps.
+/// </summary>
+public class NodeAppFixture(IMessageSink diagnosticMessageSink) : IAsyncLifetime
+{
+    private DistributedApplication? _app;
+    private string? _nodeAppPath;
+
+    public DistributedApplication App => _app ?? throw new InvalidOperationException("DistributedApplication is not initialized.");
+
+    public IResourceBuilder<NodeAppResource>? NodeAppBuilder { get; private set; }
+    public IResourceBuilder<NodeAppResource>? NpmAppBuilder { get; private set; }
+
+    public async Task InitializeAsync()
+    {
+        var builder = TestDistributedApplicationBuilder.Create();
+        builder.Services.AddXunitLogging(new TestOutputWrapper(diagnosticMessageSink));
```

Comment:
Add this to get the resource logs also, which would be useful for debugging.
```suggestion
        builder.Services.AddXunitLogging(new TestOutputWrapper(diagnosticMessageSink));
        builder.Services.AddHostedService<ResourceLoggerForwarderService>();
```